ngx-tools/src/jwt-token-managment/tokens.ts

```diff
@@ -0,0 +1,4 @@
+import { InjectionToken } from '@angular/core';
+
+export const INITIAL_TOKEN_NAME = new InjectionToken('jwt-token-managment INITIAL_TOKEN_NAME');
```

Comment:
Can we make this token a little more specific? I wouldn't be surprised if we add more tokens as we move forward.

Maybe something like `INITIAL_JWT_TOKEN_NAME`?

Reply:
Updated, FWIW this should be changable later without introducing a breaking change.